storage/plugins/viaf-authority/ViafAuthorityPlugin.php

```
@@ -0,0 +1,742 @@
+
+        $stmt = $this->db->prepare(
+            'UPDATE autori
+                SET viaf_id = ?, viaf_uri = ?, isni_id = ?, isni_uri = ?,
+                    authority_source = ?, authority_confidence = ?
+              WHERE id = ?'
+        );
+        if ($stmt === false) {
+            return $this->json($response, ['error' => true, 'message' => __('Errore interno.')], 500);
+        }
+        $stmt->bind_param('ssssssi', $viafIdParam, $viafUriParam, $isniIdParam, $isniUriParam, $sourceParam, $confParam, $id);
+        $ok       = $stmt->execute();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)